src/core/k8.ts

```diff
@@ -565,28 +565,28 @@ export class K8 {
         inputStream.pipe(inputPassthroughStream)
 
         execInstance.exec(namespace, podName, containerName, command, null, errPassthroughStream, inputPassthroughStream, false,
-          ({ status }) => self.handleCallback(status, localContext, messagePrefix))
-          .then(conn => {
-            self.logger.info(`${messagePrefix} connection established`)
-            localContext.connection = conn
+            ({ status }) => self.handleCallback(status, localContext, messagePrefix))
```

Comment:
what formatter are you using?  I noticed that `npm run format` does not do this.  I have an outstanding item to implement prettier.  although now that we agreed on google style, there is probably a better way.

In IntelliJ, I had the Palantir Style set, previously in Java, so some of those settings carried over.  If I reformat this file, it changes everything back to how I had it.  We need to all get on the same page and use the same settings.

Reply:
I usually run `npm run format` but I think in this case the change was due to a merge conflict.

Reply:
`npm run format` is not really doing whitespace until we configure prettier, which is something I was going to do soon now that we are aligned on Google code style.